concourse/pipelines/guest-package-build.jsonnet

```diff
@@ -932,31 +932,32 @@ local buildpackageimagetask = {
     },
     buildpackagejob {
       package: 'artifact-registry-yum-plugin',
-      builds: ['el7', 'el8', 'el9'],
+      builds: ['el7', 'el8', 'el8-arm64', 'el9', 'el9-arm64'],
       uploads: [
         uploadpackagetask {
           package_paths: '{"bucket":"gcp-guest-package-uploads","object":"yum-plugin-artifact-registry/yum-plugin-artifact-registry-((.:package-version))-g1.el7.noarch.rpm"}',
           universe: 'cloud-yum',
-          repo: 'yum-plugin-artifact-registry',
+          repo: 'yum-plugin-artifact-registry-el7',
         },
         uploadpackagetask {
-          package_paths: '{"bucket":"gcp-guest-package-uploads","object":"yum-plugin-artifact-registry/dnf-plugin-artifact-registry-((.:package-version))-g1.el8.noarch.rpm"}',
+          package_paths: '{"bucket":"gcp-guest-package-uploads","object":"yum-plugin-artifact-registry/dnf-plugin-artifact-registry-((.:package-version))-g1.el8.noarch.rpm"},{"bucket":"gcp-guest-package-uploads","object":"yum-plugin-artifact-registry/dnf-plugin-artifact-registry-((.:package-version))-g1.el8.aarch64.rpm"}',
```

Comment:
There shouldn't be an aarch64 rpm as this is a noarch rpm already.

Reply:
Oh actually its the other way around.... there should be two packages for EL8 and EL9 which use DNF. Instead of `g1.el8.noarch.rpm` it should be `g1.el8.x86_64.rpm` and `g1.el8.aarch64.rpm`

Reply:
yeah, we have go binaries which are arch specific, updated to use x86_64 for all el7, el8 and el9